docker-compose.yml

```diff
@@ -25,7 +25,7 @@ services:
   #   volumes:
   #     - <absolute_path_to_workspace_config>:/etc/rudderstack/workspaceConfig.json
   transformer:
-    image: "rudderlabs/develop-rudder-transformer:master"
+    image: "rudderstack/develop-rudder-transformer:master"
```

Comment:
There is no "master" tag. It should be replaced with "latest"

Reply:
Thank you. Have addressed this.